### DISCUSSION TITLE
Quick Calculations

### DISCUSSION THREAD

**Discussion by damnpenguins** [kernc/backtesting.py#679] [General]
Hi Everyone!

Hitting my head against a brick wall here. Is there an easy way to reference both the Daily Open level (say 08:00GMT) and the current Hourly open from within the next loop? I'm utilising 1m data resampled to 5M and 60M.

Struggling to get my head around pandas, numpy etc 

Im wanting to test some really simple bar based logic, ie buy the next 5M Bar if we are above the Hourly and Daily open. hold for one bar... that kind of stuff just to get my head around things....

Many thanks for your help :) 

P.S I have gone through all the discussion here and have searched StackExchange to no avail ;)

**Comment by kernc** [comment]
Why not use [`OHLCV_AGG`](https://kernc.github.io/backtesting.py/doc/backtesting/lib.html#backtesting.lib.OHLCV_AGG) and also resample to hourly/daily in `init()`. Something like:
```py
class S(Strategy):
    def init(self):
        self.hourly = self.I(lambda: self.data.df.resample('1H', label='right').agg(OHLCV_AGG).dropna())
    def next(self):
        if self.data.Close[-1] > self.hourly[-1]:
            ...

**Reply by damnpenguins** [reply]
Brilliant - thanks for that quick response!
